### ISSUE TITLE
Report screens require a stronger "Finished" indicator

### ISSUE THREAD

**Issue by gary-rowe** [bitcoin-solutions/multibit-hd#514]
User feedback from @crwatkins has indicated that several report screens would benefit from an additional "finished" indicator to ensure the process has completed and it is safe to continue. 

Some reports keep the "Next" or "Finish" buttons disabled but this can be missed so an additional message of either "Process completed successfully" (all checkmarks) or "Process completed with errors" (one or more crosses) would provide assurance.

**Comment by jim618** [comment]
Agreed we should have a 'completion' sentence like on the Wallet start up where it says 'Your wallet is ready to use'.

No need to state 'There were errors' as you can see the Xs above. It's redundant.
